megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -944,17 +944,21 @@ private void beginMyTurn() {
         tt = TurnTimer.init(this, clientgui.getClient());
     }
 
+    public void stopTimer() {
```

Comment:
Is the turn timer in a base class somewhere? If so, we may consider moving this stopTimer function further up the inheritance hierarchy so that we only have to implement it once.